src/views/ApartmentApp/index.js

```diff
@@ -260,24 +313,27 @@ export default class ApartApp extends Component {
                   <Grid container item xs={12} md={8} lg={6} direction="column" spacing={2}>
                     <Grid item>
                       <ApplicantList
+                        maxNumApplicants={MAX_NUM_APPLICANTS}
                         applicants={this.state.applicants}
                         userProfile={this.state.userProfile}
                         saving={this.state.saving}
-                        onSearchSubmit={this.onSearchSubmit}
-                        onApplicantRemove={this.onApplicantRemove}
+                        onSearchSubmit={this.handleSearchSubmit}
+                        onChangePrimary={this.handleChangePrimary}
+                        onApplicantRemove={this.handleRemove}
                         onSaveButtonClick={this.handleSaveButtonClick}
                         Authentication={this.props.Authentication}
                       />
-                      {/* <GordonDialogBox
-                        open={this.state.errorDialogOpen}
-                        onClose={this.handleCloseOkay}
-                        labelledby={'applicant-dialog'}
-                        describedby={'applicant-denied'}
-                        title={'Could Not Add Applicant'}
-                        text={this.state.errorDialogText}
-                        buttonClicked={this.handleCloseOkay}
-                        buttonName={'Okay'}
-                      /> */}
+                      <AlertDialogBox
+                        open={this.state.editDialogOpen}
+                        onClose={this.handleCloseDialog}
+                        severity={'warning'}
+                        title={'Change primary applicant?'}
+                        text={primaryApplicantAlertTest}
```

Comment:
Might need to change to "text"

Reply:
Nick caught this in his review of #911 and the change has been made 👍